README.md

```diff
@@ -93,7 +101,7 @@ cast send --private-key 0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5c
 cast balance --rpc-url http://127.0.0.1:9652/ext/bc/C/rpc 0x333d17d3b42bf7930dbc6e852ca7bcf560a69003
 ```
 
-- After calling `./scripts/local/run.sh`, you can directly send messages between the deployed subnets (in comparison, calling `./scripts/local/test.sh` runs the same setup steps, then runs the tests in `./scripts/integration-tests/` automatically). As an example, `./scripts/integration-tests/basic_send_receive.sh` can be run manually like so:
+- After calling `./scripts/local/run.sh`, you can directly send messages between the deployed subnets . As an example, `./scripts/integration-tests/basic_send_receive.sh` can be run manually like so:
```

Comment:
Thanks for digging into this. I'd actually prefer to keep the commands as is, that shell in the container persists after running `basic_send_receive.sh`, in case the user wants to run other tests or inspect the state of the chains manually. Providing a command via `-c` won't give the user an interactive shell, if I'm not mistaken.

Reply:
Good catch! Corrected now.